README.md

```diff
@@ -1,15 +1,22 @@
+# Disclaimer
+[V8](https://v8.dev/) is a community supported project. Our team at Red Hat maintains currency of the latest V8 releases to be used by Node.js on select PowerPC and S390x platforms. A list of platforms as well as their build toolchain supported by Node.js can be found here:
+https://github.com/nodejs/node/blob/master/BUILDING.md
+
+We do not provide official support on V8 if it is used outside the realm of Node.js. V8 is built and tested using the minimum compiler versions currently supported by Node.js (current versions are detailed in the above link). We do not support any other compiler versions or toolchain.
+
+If you need to report a V8 related issue specific to PowerPC or S390x platforms please open a case using the Red Hat Customer Portal https://access.redhat.com/support/cases/
```

Comment:
I wonder if we should change this to 

If you need to report an issue related issue specific to PowerPC or S390x platforms, that recreates when using Red Hat's Node.js binaries, please open a case using the Red Hat Customer Portal https://access.redhat.com/support/cases/

Or possibly drop it altogether. Seems strange to me to say "we don't support this" and then follow that with a pointer to an official support channel.

Reply:
Section is now removed.